File: FRS/DAL/Models/AuditLog.cs
using DAL.Core;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace DAL.Models
{
    public class AuditLog
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public long Id { get; set; }
        public int? InstitutionId { get; set; }
        public int? UserId { get; set; }
        public string UserName { get; set; }
        public string InstitutionName { get; set; }

        public DateTime EventDateTime { get; set; }

        public AuditLogType LogType { get; set; }

        [Required]
        [MaxLength(256)]
        public string TableName { get; set; }

        [Required]
        [MaxLength(256)]
        public string RecordId { get; set; }
        public string GroupId { get; set; }
        public string ActionName { get; set; }
        public string Remarks { get; set; }

        public virtual ICollection<AuditLogDetail> Details { get; set; }
    }
}

File: FRS/FRS/ClientApp/src/app/models/audit-log.ts

export class AuditLog {
  public id: string;
  public userName: string;
  public eventDateTime: string;
  public logType: string;
  public tableName: string;
  public recordId: string;
}

export class UserActivityLog {
  
  public groupId: string;
  public userName: string;
  public actionName: string;
  public auditLogId: string;
  public eventDateTime: string;
  public recordId: string;
  public logType: string;
  public remarks: string;
  public propertyName: string;
  public oldVal: string;
  public newVal: string;
  public detailRemarks: string;
}
